plugins/feeds/public/misp.py

```diff
@@ -171,7 +171,8 @@ def analyze(self, event, instance):
         if external_analysis:
             context['external sources'] = '\r\n'.join(external_analysis)
 
-        if 'galaxy_filter' not in self.instances[instance]:
+        if 'galaxy_filter' not in self.instances[instance] \
```

Comment:
I would suggest something easier to read & understand, like:
```Python
         if not self.instances[instance].get('galaxy_filter'):
```

Reply:
I do not usually use this function

Reply:
Yep good remark,
I do not usually use this function.

Thanks